bpf/lib/nodeport.h

```diff
@@ -1006,13 +1001,13 @@ static __always_inline int rev_nodeport_lb6(struct __ctx_buff *ctx, __u32 *ifind
 		}
 	}
 
-	return CTX_ACT_OK;
+	return CTX_ACT_REDIRECT;
 }
 
 __section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_IPV6_NODEPORT_REVNAT)
 int tail_rev_nodeport_lb6(struct __ctx_buff *ctx)
 {
-	int ext_err = 0, ret = 0;
+	int ext_err = 0, ret = 0, ret2 = 0;
```

Comment:
Nit: `ret2` could be defined in the `if { }` block, and doesn't have to be initialised